main.tf

```diff
@@ -183,3 +184,20 @@ resource "tfe_sentinel_policy" "gcp-restrict-machine-type" {
   policy       = "${file("./gcp-restrict-machine-type.sentinel")}"
   enforce_mode = "soft-mandatory"
 }
+
+# Policy that requires modules to come from Private Module Registry
+data "template_file" "require-modules-from-pmr" {
+  template = "${file("./require-modules-from-pmr.template")}"
+
+  vars {
+    organization = "${var.tfe_organization}"
+  }
+}
+
+resource "tfe_sentinel_policy" "require-modules-from-pmr" {
+  name         = "require-modules-from-pmr"
+  description  = "Require all modules to come from the Private Module Registy of the current org"
+  organization = "${var.tfe_organization}"
+  policy       = "${data.template_file.require-modules-from-pmr.rendered}"
+  enforce_mode = "hard-mandatory"
```

Comment:
Do you think maybe `soft-mandatory` might be more realistic, as a safety valve for testing modules that are in review and haven't been approved into the private registry yet?